document_merge_service/api/tests/test_encrypt_templates.py

```diff
@@ -0,0 +1,64 @@
+from io import StringIO
+
+from django.core.files import File as DjangoFile
+from django.core.management import call_command
+from storages.backends.s3 import S3Storage
+
+from document_merge_service.api.data import django_file
+
+
+def test_encrypt_templates(db, settings, mocker, template_factory):
+    template_factory(template=django_file("docx-template.docx"))
+
+    settings.DMS_ENABLE_AT_REST_ENCRYPTION = True
+    settings.STORAGES["default"]["BACKEND"] = "storages.backends.s3.S3Storage"
+    settings.STORAGES["default"]["OPTIONS"] = settings.S3_STORAGE_OPTIONS
+    settings.STORAGES["default"]["OPTIONS"]["object_parameters"] = {
+        "SSECustomerKey": "x" * 32,
+        "SSECustomerAlgorithm": "AES256",
+    }
```

Comment:
Those nested settings will probably leak to other tests at some point.

You should either monkeypatch those or make copy that will be lost after the test.

Reply:
changed to copy